requirements.txt

```diff
@@ -1,6 +1,33 @@
-matplotlib==3.0.2
-PySide==1.2.4
-# PySide2==5.12.0
-Shiboken==1.2.2
-six==1.12.0
-Markdown==3.2.2
+area==1.1.1
+bzr_pqm==1.4.0
+cog==0.6.1
+ConfigParser==5.3.0
+defusedxml==0.7.1
+GitPython==3.1.31
+ifcopenshell==0.7.0.230318
+kdtree==0.16
```

Comment:
where is this used?

Reply:
This is the only python file using it [src/3rdParty/libkdtree/python-bindings/py-kdtree_test.py](https://github.com/FreeCAD/FreeCAD/blob/5d49bf78de785a536f941f1a6d06d432582a95d3/src/3rdParty/libkdtree/python-bindings/py-kdtree_test.py)
Probably can be omitted.